tests/test_data/output_step2.smi

```diff
@@ -1,1976 +1,1976 @@
-C1(c2cc3cc(C)ccc3nc2Oc2ccc(Cl)cc2)C(CC#N)=C(N)Oc2c1c(=O)oc1c2cccc1
-C(=O)(O)CC(c1ccccc1)C(C(=O)O)N
-c12ccc(CCCCC3C(CCC)OC3=O)cc1cccc2
-c1c2c(nccn2)c(NCC2CNCC(F)(F)C2)nc1-c1cn(C)nc1
-Clc1ccc(C2N(CC(NCCCN3CCOCC3)=O)C(=O)C(CC(NCc3c(Cl)cc(Cl)cc3)=O)S2)c(Cl)c1
-C1CN(Cc2ccc(Cl)cc2)CCC1Nc1nc(Oc2c(C)cc(C=CC#N)cc2C)c2c(ccs2)n1
-c1ccoc1C(C)=NNC(=S)NC(C)C
-c1cc2c(cc1)[nH]c(C(N1CC(Oc3ncccc3C3=CCOCC3)C1)=O)n2
-c12c(ccc(C)c1)n1c(C)nn(Cc3cc(C)ccc3)c(=O)c1c2
-C1CNCCC1n1cnc(-c2ccc(F)cc2)c1-c1nc(NC(CO)c2ccccc2)ncc1
-C1(CC=CCCC(=O)O)COC(c2ccc(C#N)cc2)OC1c1c(O)cccc1
-C1CN(C)CCN1CCCN=C(N)N
-N(N=C(C(=N)SCc1ccccc1)C#N)c1ccc(OC)cc1
-c1c2c(ccc1Br)C(=O)C(=C1C(=O)Nc3ccccc31)N2
-C(CC)(Nc1c(O)ccc(NC(C=Cc2ccccc2)=O)c1)=O
-c1(-c2ccncc2)c(C=CC(N2CCN(CCOCCO)CC2)=O)cn(-c2ccccc2)n1
-N(C(COc1c(OC)cccc1)=O)c1c(C(N2CCCCC2)=O)cccc1
-C1(=O)C(C#N)C(c2c(OC)cccc2)C(C#N)=C(SCC(=O)OCCCC)N1
-c12c(SSc3c4n(CC)ccc(=O)n4nc3-c3ccccc3)c(-c3ccccc3)nn1c(=O)ccn2CC
-O1CCN(c2ncnc(Oc3ccc(NC(=O)Nc4cc(C(C)(C)C)on4)cc3F)c2)CC1
-C1C(CCNC(c2sccc2)=O)N(C)c2ccc(Cl)cc2C(c2ccccc2)=N1
-O(P(=O)(NCCCl)NCCCl)Cc1cnc([N+]([O-])=O)n1C
-N1(Cc2ccc(NC(c3cc(OC)cc(OC)c3)=O)cc2)CCCCC1
-C(C=Cc1ccc(Oc2ccc(Cl)cc2)cc1)(=O)c1c(O)c(C(O)=O)cc(Br)c1
-C(CCOc1c(CN)c(OCCCO)ccc1)O
-O=C(NC(C(=O)NC(C(NC(Cc1c[nH]c2c1cccc2)C(=O)Nc1c(C(=O)N)cccc1)=O)CCCN=C(N)N)Cc1ccccc1)C1(NC(NCCCC)=O)CCC(c2ccccc2)CC1
-CCN(C(C(CC(C)C)S(c1nn(C(N(CC)CC)=O)cn1)(=O)=O)=O)CC
-COCc1ccc(C(=O)N2CCCC(C(OCC)=O)(Cc3c(C)cccc3)C2)o1
-C1(=O)N(CC(=O)O)C(=O)SC1=Cc1cc(O)c(OC)cc1
-C1C2(C3SCC(=O)N3c3ccncc3)CC3CC(C2)CC1C3
-c1ccc(Oc2ccccc2)cc1C(=O)Nc1cc(Cl)ccc1
-c12c3c(c4c5c(n6c4c1n(c1ccccc21)CC(CO)C6)ccc(Br)c5)C(=O)NC3
-c1(CC(=O)N2CCSCC2)ccc(NCc2nc(CC)no2)cc1
-C(CC(C)C)(NP(=O)(Oc1ccccc1)OC)C(N)=O
-C1C(S(c2ccccc2)(=O)=O)CCN(Cc2ccccc2)C1
-CCCCCOc1ccc2c(c1)c(CCN)c[nH]2
-O=S(=O)(c1c(C)cc(C)cc1C)NCc1ccc(C(NCCN2Cc3ccccc3C2)=O)cc1
-c1(C)nc2n(cccc2)c1C(OCC)=O
-N(CCc1ccccc1)(C1CCN(CCC)CC1)Cc1ccccc1
-N(C(Cc1ccccc1)C(OC)=O)C(=O)CCc1[nH]c(=O)c2c(n1)cccc2
-N12CCC3(CC1c1c(cccc1)CC2)OC(=O)NC3
-c1(N)cc(O)nc(SCCCCC)n1
+C(c1c2ccccc2[nH]c1)C(NN=Cc1occc1)=O
```

Comment:
Let's not change this file unless we have to.

Reply:
Reverted it back to the original one.